src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -184,7 +184,10 @@
                                                      [30 125]
                                                      [14 0]
                                                      header-extrapolation-option)]
-    [rn/view {:flex 1}
+    [rn/view
+     (cond-> {:flex 1}
+       platform/android?
+       (assoc :scaleY -1))
```

Comment:
why `cond->` though? isn't it easier to do something like `{:flex 1 :scaleY (if platform/android? -1 1)}`?

(same goes for everything else below)

Reply:
Well technically the nesting level is the same, the number of words is the same. One pair of brackets instead of another magic value. I see very little difference and I prefer `cond->` here.

Reply:
just extracted it to a separate fn